cirq/google/xmon_gates.py

```diff
@@ -213,18 +212,11 @@ def to_proto_dict(self, *qubits):
         }
         return {'exp_11': exp_11}
 
-    def try_cast_to(self, desired_type, ext):
-        if desired_type is ops.KnownMatrix and self.has_matrix():
-            return self
-        return super().try_cast_to(desired_type, ext)
-
-    def has_matrix(self):
-        return not isinstance(self.half_turns, value.Symbol)
-
-    def matrix(self):
-        if not self.has_matrix():
-            raise ValueError("Don't have a known matrix.")
-        return protocols.unitary(ops.Rot11Gate(half_turns=self.half_turns))
+    def _unitary_(self) -> Optional[np.ndarray]:
```

Comment:
I thought the plan was to change the return type here to `Union[np.ndarray, type(NotImplemented)]`

Reply:
Good catch. Done.